TcUnit-Runner/Program.cs

```diff
@@ -34,6 +34,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using TCatSysManagerLib;
+using TwinCAT.Ads;
```

Comment:
What's the licensing to use it? Do we need to add a license-file that should be included in the source + deployment? (compare to log4net)

Reply:
It is an official Beckhoff library that is licensed under MIT. So, I guess a license notice is sufficient.

Reply:
Sounds good!